kuzzleio/documentation
# [fix] images in koncorde filter terms

## What does this PR do?
Add some missing images

### How should this be manually tested?
Visit /core/1/koncorde/essentials/terms/#geoboundingbox and ensure the images are present.
